packages/babel-traverse/src/hub.js

```diff
@@ -1,6 +1,7 @@
-export default class Hub {
-  constructor(file, options) {
-    this.file = file;
-    this.options = options;
-  }
+export type Hub = {
+  mark: (type: string, message: string) => void;
+  addHelper: (name: string) => Object;
+  getScope: () => Object;
+  getCode: () => string;
+  buildError:(node: Object, msg: string, Error: Error) => Error;
```

Comment:
yeah, totally! I think using `export default class Hub ` instead `export type Hub` would probably be good here.
Though `Hub` itself wasn't been referred in repo, it is possible that someone used `Hub` to store states. We can totally keep it.